public/app/features/serviceaccounts/ServiceAccountTokensTable.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC } from 'react';
+import { DeleteButton, Icon, IconName, Tooltip, useTheme2 } from '@grafana/ui';
+import { dateTimeFormat, GrafanaTheme2, TimeZone } from '@grafana/data';
+
+import { ApiKey } from '../../types';
+import { css } from '@emotion/css';
+
+interface Props {
+  apiKeys: ApiKey[];
+  timeZone: TimeZone;
+  onDelete: (apiKey: ApiKey) => void;
+}
+
+export const ServiceAccountTokensTable: FC<Props> = ({ apiKeys, timeZone, onDelete }) => {
+  const theme = useTheme2();
+  const styles = getStyles(theme);
+
+  return (
+    <>
+      <h3 className="page-heading">Tokens</h3>
+      <table className="filter-table">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Expires</th>
+            <th style={{ width: '34px' }} />
+          </tr>
+        </thead>
+        {apiKeys.length > 0 ? (
+          <tbody>
+            {apiKeys.map((key) => {
+              const isExpired = Boolean(key.expiration && Date.now() > new Date(key.expiration).getTime());
```

Comment:
In JS/TS it's easier :)
```suggestion
              const isExpired = key.expiration && Date.now() > new Date(key.expiration).getTime();
```